### PR TITLE
Create Article Schema

### PR DESCRIPTION
**Pull request by cathysnider**

### CODE REVIEW

File: schema_definitions/article.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+
+## Schema.org definition link : https://schema.org/Article
+
+ | Field Display Name | Multivalue | Required | Data Structure | Schema.org Field Name | Express Field Name | Express Data Structure | Notes                                 |
+ | :----------------: | :--------: | :------: | :------------: | :-------------------: | :----------------: | :--------------------: | ------------------------------------- |
+ |Title | N | Y | text | Content Title | Title | Text | used by pathauto for generating URL | 
+ |Body | N | N | Long text | articleBody |  |  |  | 
+ |Summary | N | N | Text | abstract |  |  |  | 
+ |Date Display | N | Y | list (text) |  -  |  |  | A dropdown to show or hide date. | 
+ |Image: Photo | Y | N | Image | image |  |  |  |
```
Comment (jcsparks) [inline schema_definitions/article.md:15]: We won't be using the Image type in D9, the Media module is in core now and fully matured to the point where there's no reason not to use Media for digital files wherever possible.

File: schema_definitions/article.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+## Schema.org definition link : https://schema.org/Article
+
+ | Field Display Name | Multivalue | Required | Data Structure | Schema.org Field Name | Express Field Name | Express Data Structure | Notes                                 |
+ | :----------------: | :--------: | :------: | :------------: | :-------------------: | :----------------: | :--------------------: | ------------------------------------- |
+ |Title | N | Y | text | Content Title | Title | Text | used by pathauto for generating URL | 
+ |Body | N | N | Long text | articleBody |  |  |  | 
+ |Summary | N | N | Text | abstract |  |  |  | 
+ |Date Display | N | Y | list (text) |  -  |  |  | A dropdown to show or hide date. | 
+ |Image: Photo | Y | N | Image | image |  |  |  | 
+ |Image: Thumbnail | N | N | image | image |  |  |  |
```
Comment (jcsparks) [inline schema_definitions/article.md:16]: We may not need a separate image for thumbnail either... it's possible that Media has this included for image file types.

File: schema_definitions/article.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+## Purpose : for time-sensitive content like news, announcements or blog posts
+
+## Schema.org definition link : https://schema.org/Article
+
+ | Field Display Name | Multivalue | Required | Data Structure | Schema.org Field Name | Express Field Name | Express Data Structure | Notes                                 |
+ | :----------------: | :--------: | :------: | :------------: | :-------------------: | :----------------: | :--------------------: | ------------------------------------- |
+ |Title | N | Y | text | Content Title | Title | Text | used by pathauto for generating URL | 
+ |Body | N | N | Long text | articleBody |  |  |  | 
+ |Summary | N | N | Text | abstract |  |  |  | 
+ |Date Display | N | Y | list (text) |  -  |  |  | A dropdown to show or hide date. |
```
Comment (jcsparks) [inline schema_definitions/article.md:14]: Does this just default to the content publish date?  Wouldn't it be more flexible to he an actual datetime field here... we wouldn't need a display flag then, if it's filled out we show it, if it's not then we don't.

**Review by jcsparks — CHANGES_REQUESTED** [review]
Images should be moved to Media for sure... need to look at datetime field as well.

File: schema_definitions/article.md (hunk 3 above)
Comment (cathysnider) [reply schema_definitions/article.md:14]: That would be one way of solving the problem. We also offer the option to schedule publication; perhaps we'd want to tie these two functionalities together in some way.

File: schema_definitions/article.md (hunk 2 above)
Comment (cathysnider) [reply schema_definitions/article.md]: People do upload several images to their articles sometimes. We'll need a way for them to mark which image is to be used as the thumbnail.

**Review by jcsparks — APPROVED** [review]
Looks like a good start.  Might want to make a more advanced new story content type later on.

**Pull request merged**